native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/internal/Utils.java

```diff
@@ -38,70 +38,19 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-package org.graalvm.buildtools;
-
-import java.io.File;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.Optional;
-import java.util.regex.Pattern;
-import java.util.stream.Stream;
+package org.graalvm.buildtools.gradle.internal;
 
 /**
  * Utility class containing various native-image and JVM related methods.
  * Keep this file in sync across all build tool plugins.
  */
 public class Utils {
-    public static final boolean IS_WINDOWS = System.getProperty("os.name", "unknown").contains("Windows");
-    public static final String EXECUTABLE_EXTENSION = (IS_WINDOWS ? ".exe" : "");
-    public static final String NATIVE_IMAGE_EXE = "native-image" + (IS_WINDOWS ? ".cmd" : "");
+    private static final boolean IS_WINDOWS = System.getProperty("os.name", "unknown").contains("Windows");
+    private static final String EXECUTABLE_EXTENSION = (IS_WINDOWS ? ".cmd" : "");
+    public static final String NATIVE_IMAGE_EXE = "native-image" + EXECUTABLE_EXTENSION;
     public static final String NATIVE_IMAGE_OUTPUT_FOLDER = "native";
     public static final String AGENT_PROPERTY = "agent";
-    public static final String AGENT_OUTPUT_FOLDER = Paths.get(NATIVE_IMAGE_OUTPUT_FOLDER, "agent-output").toString();
-    public static final String NATIVE_TESTS_EXE = "native-tests" + EXECUTABLE_EXTENSION;
+    public static final String AGENT_OUTPUT_FOLDER = NATIVE_IMAGE_OUTPUT_FOLDER + "/agent-output";
```

Comment:
By looking at the Gradle source code it seems to me that this would work for Windows as well (since it should internally use `Project#file()`) but I'm not 100% sure?

Reply:
I'm not sure what you're referring to? The fact that it includes a `/`? If so, yes, it works under Windows too.